data/data-pipeline/data_pipeline/etl/score/etl_utils.py

```diff
@@ -48,3 +51,61 @@ def check_score_data_source(
                 "No local score tiles data found. Please use '-d aws` to fetch from AWS"
             )
             sys.exit()
+
+
+def floor_series(series: pd.Series, number_of_decimals: int) -> pd.Series:
+    """
+    Floors all non-null numerical values to a specific number of decimal points
```

Comment:
Nit: can we move the summary to the first line, like `"""Floors all non-null numerical...`? 

See for reference https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html